public/scripts/frames.js

```diff
@@ -62,61 +63,97 @@ function removeInstance(data) {
 
 function setDataIntoCard() {
 
-    document.getElementById("cardsList").innerHTML = "";
+    //document.getElementById("cardsList").innerHTML = "";
 
     dataset.forEach(dataObj => {
-        var title = dataObj.title;
-        var priority = dataObj.priority;
-        var assignee = dataObj.assignee;
-        var creator = dataObj.creator;
-
-        var tickets = document.getElementById("cardsList").innerHTML;
-
-        var frame = "<div class=\"card text-white bg-dark\" style=\"max-width: 32rem;\">";
-
-        frame += "<div class=\"card-header\"> XX/XX/XXXX </div>";
-        frame += "<div class=\"card-body py-3\">";
-        frame += "<div class=\"row pb-2\">";
-        frame += "<div class=\"col-md-4 col-4\">";
-        frame += "<h6 class=\"card-title text-center py-1\" style=\"background-color: black;\"> Id: </h6>";
-        frame += "</div>";
-        // frame += "<div class=\"col-md-8 col-8 px-2\">" + id + "</div>";
-        frame += "</div>";
-
-        frame += "<div class=\"row pb-2\">";
-        frame += "<div class=\"col-md-4 col-4\">";
-        frame += "<h6 class=\"card-title text-center py-1\" style=\"background-color: black;\"> Title: </h6>";
-        frame += "</div>";
-        frame += "<div class=\"col-md-8 col-8 px-2\">" + title + "</div>";
-        frame += "</div>";
-
-        frame += "<div class=\"row pb-2\">";
-        frame += "<div class=\"col-md-4 col-4\">";
-        frame += "<h6 class=\"card-title text-center py-1\" style=\"background-color: black;\"> Priority: </h6>";
-        frame += "</div>";
-        frame += "<div class=\"col-md-8 col-8 px-2\">" + priority + "</div>";
-        frame += "</div>";
-
-        frame += "<div class=\"row pb-2\">";
-        frame += "<div class=\"col-md-4 col-4\">";
-        frame += "<h6 class=\"card-title text-center py-1\" style=\"background-color: black;\"> Assignee: </h6>";
-        frame += "</div>";
-        frame += "<div class=\"col-md-8 col-8 px-2\">" + assignee + "</div>";
-        frame += "</div>";
-
-        frame += "<div class=\"row pb-2\">";
-        frame += "<div class=\"col-md-4 col-4\">";
-        frame += "<h6 class=\"card-title text-center py-1\" style=\"background-color: black;\"> Creator: </h6>";
-        frame += "</div>";
-        frame += "<div class=\"col-md-8 col-8 px-2\">" + creator + "</div>";
-        frame += "</div>";
-
-        frame += "</div>";
-        frame += "</div>";
-        tickets += frame;
-
-        document.getElementById("cardsList").innerHTML = tickets;
+        console.log(dataObj.status);
+        frames(dataObj);
```

Comment:
I guess filter method will be called from here and from filter, frames method will be called.